data/tlg0341/tlg002/__cts__.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts"
+    groupUrn="urn:cts:greekLit:tlg0341"
+    urn="urn:cts:greekLit:tlg0341.tlg002"
+    xml:lang="grc"
+>
+    <!--
+      Work must have at least one title node.
+      Title node needs xml:lang declaration, it reflects the language of the title.
+    -->
+    <ti:title xml:lang="eng">Alexandra</ti:title>
+    <ti:title xml:lang="grc">Ἀλεξάνρα</ti:title>
+    <!--
+      For each "text", either edition, translation, or commentary, there should be a ti:edition,
+    ti:translation, or ti:commentary node
+
+      The edition nodes has two attributes :
+        - The first one, workUrn, contains only the urn up to the work component
+        - The second, urn, contains the full urn
+    -->
+    <ti:edition
+        workUrn="urn:cts:greekLit:tlg0341.tlg002"
+        urn="urn:cts:greekLit:tlg0341.tlg002.perseus-grc1"
+    >
+        <!--
+          Edition, Translation, and Commentary must have at least one label node.
+          Label represents the title of the edition.
+          Label node needs xml:lang declaration, it reflects the language of the title.
+        -->
```

Comment:
hi @lcerrato  and @pletcher I am happy to offer suggestions, and also to fix cts_.xml and other bibliographic metadata for all these files as well.

Reply:
@pletcher @AlisonBabeu 
I would recommend this file be corrected now in this PR so that we have an example. 
Let's also remove all comments and soft wrap the lines for easier reading.